src/player.js

```diff
@@ -322,6 +322,9 @@ export default class Player extends FakeEventTarget {
     this._createPlayerContainer();
     this._appendPosterEl();
     this.configure(config);
+    if (config.logLevel && this.LogLevel[config.logLevel]){
```

Comment:
Why can't we do this in the `loadPlayer` factory function of playkit.js before the initialization of the player?
in configure we already write logs and if we are doing it after it won't affect this, no?

Reply:
I see now that we are using this for the check, but we can define fallback level in such cases and avoid this

Reply:
If you refer to the version and name printing in index.js, we don't want to print them and in any case these printouts are in load time before CTOR is called, and setting the config level can be done only in init time

Reply:
No, I'm not referring to the version and name printouts.
I'm mean that you set the log level too late - after the configure method.
The configure method flow will print logs and seems like those logs won't have the correct log level. 
Don't you think the correct place to handle the log level is before creating the player? in the loadPlayer method?

Reply:
No, the static API should remain simple IMHO.
Constructor is where everything should be setup, and in the case of logs, this line should just some before others if it's more important.
Fixed by moving to be first in line